sort-python-dictionary/compare_lookup_dict_vs_list.py

```diff
@@ -0,0 +1,38 @@
+from timeit import timeit
+
+lookups = [15, 18, 19, 16, 6, 12, 5, 3, 9, 20, 2, 10, 13, 17, 4, 14, 11, 7, 8]
+
+list_setup = """
+from samples import list_of_dictionaries
```

Comment:
In the tutorial you have this import statement at the top of the file. Does that matter? :thinking:

Reply:
Right, this was a linter issue. Linter was saying that I had imports that I wasn't using 😕